testsuite/integration/microprofile/pom.xml

```diff
@@ -385,7 +385,6 @@
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>testcontainers</artifactId>
-            <scope>test</scope>
```

Comment:
Why was the scope removed?

Reply:
Not sure. That might be a wart from an experiment.